arm-build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Do we really need another script verse just adding some conditions for arm in the existing scripts?

Reply:
Removed, this was due to difference between my local machine and CI.